-------------------------------------------------------------------------------
Test set: com.testassignment04.AccountTest
-------------------------------------------------------------------------------
Tests run: 4, Failures: 1, Errors: 2, Skipped: 0, Time elapsed: 0.036 s <<< FAILURE! - in com.testassignment04.AccountTest
testDeposit_with_zero_negative_amount  Time elapsed: 0.024 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: Unexpected exception type thrown ==> expected: <java.lang.IllegalArgumentException> but was: <java.lang.NullPointerException>
	at com.testassignment04.AccountTest.testDeposit_with_zero_negative_amount(AccountTest.java:43)
Caused by: java.lang.NullPointerException
	at com.testassignment04.AccountTest.lambda$testDeposit_with_zero_negative_amount$0(AccountTest.java:43)
	at com.testassignment04.AccountTest.testDeposit_with_zero_negative_amount(AccountTest.java:43)

testWithdraw_with_invalid_amount  Time elapsed: 0.002 s  <<< ERROR!
java.lang.NullPointerException
	at com.testassignment04.AccountTest.testWithdraw_with_invalid_amount(AccountTest.java:49)

testDeposit_with_positive_amount  Time elapsed: 0.001 s  <<< ERROR!
java.lang.NullPointerException
	at com.testassignment04.AccountTest.testDeposit_with_positive_amount(AccountTest.java:36)

